### PR TITLE
Get Recent Submitted Problem status of user and get contest status APIs added

### PR DESCRIPTION
**Pull request by gangadhargo**
Get Recent Submitted Problem status of user and get contest status APIs added

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -146,11 +146,52 @@ export default class Cf {
...
+            }).catch(function(error){
+                throw new Error('Not able to retrieve Contest Status')
+            });
+        return contest.result;
+    };
 }
+
+
+
+
```
Comment (AbdelH2O) [inline src/index.js]: No need for these empty lines.
```suggestion
```

**Review by AbdelH2O — COMMENTED** [review]
Good job adding these 2 endpoints!
Thanks for the contribution!
Let's merge it once the empty lines are removed.

**Comment by gangadhargo** [comment]
Thank you, do you have complete action items for the project I like to contribute further :)

between removed the empty lines

**Comment by AbdelH2O** [comment]
> Thank you, do you have complete action items for the project I like to contribute further :)
> 
> between removed the empty lines

I built this wrapper with the minimum functionality I needed for another project (which relies on cf problems as you can imagine). So no, I don't exactly have a plan of action. But if you're interested in contributing to either one of them (this wrapper or the other project) hit me up at a.hana@aui.ma to get in touch!

**Review by AbdelH2O — APPROVED** [review]
A new line at the end would be most welcome hh

**Comment by gangadhargo** [comment]
sure 👍🏿 . what is the other project?

**Comment by AbdelH2O** [comment]
> sure 👍🏿 . what is the other project?

It's related to lockout contests. Still working on a name.

**Pull request merged**